{
  Iterable<ResourceNotFoundException> throwables=Iterables.filter(Throwables.getCausalChain(from),ResourceNotFoundException.class);
  if (Iterables.size(throwables) >= 1) {
    return false;
  }
 else {
    return !rto404.apply(from);
  }
}
